mango/dashboards/file_explorer_app.py

```diff
@@ -519,6 +525,24 @@ def _render_file_content(self, path_selected: str, key: str):
                         df = pd.read_excel(excel_file, sheet_name=key_tab)
                         st.dataframe(df, use_container_width=True)
 
+        elif path_selected.endswith(".json"):
+            with st.spinner("Wait for it..."):
+                with open(path_selected, "r") as f:
+                    data = json.load(f)
+                    st.checkbox("As table", value=False, key=f"json_df_{key}")
+                    if st.session_state[f"json_df_{key}"]:
+                        sheets = list(data.keys())
+                        list_of_tabs = st.tabs(sheets)
+                        dict_of_tabs = {}
+                        for i, tab in enumerate(list_of_tabs):
+                            dict_of_tabs[sheets[i]] = tab
+                        for key_tab, tab in dict_of_tabs.items():
+                            with tab:
+                                df = pd.DataFrame.from_dict(data[key_tab])
+                                st.dataframe(df, use_container_width=True)
```

Comment:
in order to be a little more robust, you could do:
```
with tab:
    try:
        df = pd.DataFrame.from_dict(data[key_tab])
        st.dataframe(df, use_container_width=True)
except Exception:
        st.json(data[key_tab])
```
(putting the right exception, probably TypeError)

Reply:
I have put the try except but without getting an error, I prefer to get a warning in an interface but you can still use the App.

Reply:
yes, I didn't mean raising an error but putting the right type of error instead of except Exception.
I think except TypeError might be the right one but it may be an other error.